six/modules/c++/six.sicd/source/Utilities.cpp

```diff
@@ -489,6 +489,24 @@ void Utilities::getWidebandData(
             offset, extent, buffer);
 }
 
+Vector3 Utilities::getGroundPlaneNormal(const ComplexData& data)
+{
+    Vector3 groundPlaneNormal;
+
+    if (data.radarCollection->area.get() &&
+        data.radarCollection->area->plane.get())
+    {
+        AreaPlane areaPlane = *data.radarCollection->area->plane;
+        groundPlaneNormal = cross(areaPlane.xDirection->unitVector,
```

Comment:
Hmm, I'm actually confused how the compiler knows how to resolve that... I don't see a `using namespace math::linear` anywhere.  Anyway yeah, still stick the `math::linear::` in front of it to be explicit.